src/heretic/model.py

```diff
@@ -202,11 +208,14 @@ def abliterate(
                     layer_refusal_direction,
                     layer_refusal_direction,
                 ).to(self.model.dtype)
-
+                
                 for matrix in matrices:
                     # In-place subtraction is safe as we're not using Autograd.
```

Comment:
This comment should be moved down so it remains attached to the line it refers to (`matrix.sub_`).